server/datastore/mysql/migrations/tables/20220815184143_AddOrbitNodeKeyToHosts.go

```diff
@@ -0,0 +1,21 @@
+package tables
+
+import (
+	"database/sql"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20220815184143, Down_20220815184143)
+}
+
+func Up_20220815184143(tx *sql.Tx) error {
+	_, err := tx.Exec(`ALTER TABLE hosts ADD COLUMN orbit_node_key VARCHAR(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin DEFAULT NULL;`)
+	return err
+}
+
+func Down_20220815184143(tx *sql.Tx) error {
```

Comment:
No need to define `Down_*`. Leave empty. At some point they were used, but then decided to not use such feature.

PS: We should get rid of all `Down_*` methods eventually to avoid confusion.